daprdocs/content/en/reference/components-reference/supported-pubsub/setup-pulsar.md

```diff
@@ -62,6 +62,11 @@ spec:
       }
 ```
 
+{{% alert title="Warning" color="warning" %}}
+This component supports storing the `token` parameter and any other sensitive parameter and data as Kubernetes Secrets.
+{{% /alert %}}
+
```

Comment:
The general warning we have is this
"The above example uses secrets as plain strings. It is recommended to use a secret store for the secrets as described [here](https://docs.dapr.io/operations/components/component-secrets/)."

I would use this ^^ combining this with the " storing the `token` parameter and any other sensitive parameter" as  specific to this component

Reply:
@msfussell pushed an update - let me know if that works?